__tests__/tests.js

```diff
@@ -0,0 +1,41 @@
+const puppeteer = require("puppeteer");
+const { beforeAll, expect, test } = require("@jest/globals");
+
+beforeAll(async () => {
```

Comment:
Here, we eagerly check that the developer ran `npm start` in another tab already. That allows us to give a good error message if they forgot to.